test/box-luatest/gh_7362_net_box_async_ping_test.lua

```diff
@@ -0,0 +1,40 @@
+local net = require('net.box')
```

Comment:
For features (this seems to be a feature) we don't use `gh_xxxx` prefix in test file name. It is only for bug tests.

Reply:
I'll keep that in mind for the future